app/controllers/training_controller.rb

```diff
@@ -60,7 +59,10 @@ def add_training_root_breadcrumb
   end
 
   def add_library_breadcrumb
-    add_breadcrumb params[:library_id].titleize, :training_library_path
+    add_breadcrumb(
+      TrainingLibrary.find_by(slug: params[:library_id])&.translated_name || params[:library_id],
+      :training_library_path
+    )
```

Comment:
Hmm... This change is not what we want for Wiki Education Dashboard, at least. The current behavior, where the breadcrumb is the short version based on the library slug, is better than using the full library name:



Current:
    Training Library > Students > Sandboxes, talk pages, and watchlists

New:
    Training Library > Student Training Modules > Sandboxes, talk pages, and watchlists

However, using the translations is definitely what we want on P&E Dashboard. Can you think of any better solution that just switching based on `Features.wik_ed?`